cogs/whisper.py

```diff
@@ -12,14 +12,15 @@ def __init__(self, bot):
     def cog_unload(self):
         self.bot.loop.create_task(self.session.detach())
 
-    @commands.Cog.listener()
-    async def on_message(self, message):
-        # channel id for public server
+    @commands.dm_only()
+    @commands.command(aliases=["whisper"])
+    async def get_whisper(self, ctx, *message: str):
```

Comment:
would `send` be more appropritate prefix here?

also noting, that when `!help` is called, `get_whisper` is the name that will be shown, would you prefer that or `whisper`, at which case, you'd need to swap aliases.

Reply:
I think 'send' should be reserved for a future command, what do you think?
I added some help text to the function, and changed the function name to 'psst'.

Reply:
I think 'send' should be reserved for a future command, what do you think?
I added some help text to the function, and changed the function name to 'psst'.

Reply:
I think 'send' should be reserved for a future command, what do you think?
I added some help text to the function, and changed the function name to 'psst'.

Reply:
thats up to you, i just think `get` isn't representitive of this function's function.